controllers/articles.js

```diff
@@ -153,6 +154,127 @@ export default class ArticleController {
     }
   }
 
+  /**
+   * @desc create a reaction
+   * @param {Object} articleSlug, the response obje
+   * @param {Object} userId - the response object
+   * @param {Object} type - the response object
+   * @memberof ArticleController
+   * @return {Object} returns an object
+   */
+  static async createReaction(articleSlug, userId, type) {
```

Comment:
@Dubby20 you can do pass in a default value for like: `static async createReaction(articleSlug, userId, like = true)`. So that way, you don't need the ternary operator. You can pass then pass in false when disliking.
That way we don't need the eslint thingy up there too.

Reply:
Okay. Thanks